src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -13,24 +13,38 @@ public abstract partial class CriticalHandleMinusOneIsInvalid : System.Runtime.I
         protected CriticalHandleMinusOneIsInvalid() : base(System.IntPtr.Zero) { }
         public override bool IsInvalid { [System.Security.SecurityCriticalAttribute]get { throw null; } }
     }
+
     [System.Security.SecurityCriticalAttribute]
     public abstract partial class CriticalHandleZeroOrMinusOneIsInvalid : System.Runtime.InteropServices.CriticalHandle
     {
         protected CriticalHandleZeroOrMinusOneIsInvalid() : base(System.IntPtr.Zero) { }
         public override bool IsInvalid { [System.Security.SecurityCriticalAttribute]get { throw null; } }
     }
+
```

Comment:
All of these blank lines will likely be removed the next time someone uses a tool to generate this file.  Probably best to just not add them.